SkillChat.Interface/LogOn.cs

```diff
@@ -8,6 +8,12 @@ public class LogOn : IClientMethod
         public string Id { get; set; }
         public string UserLogin { get; set; }
         public DateTimeOffset ExpireTime { get; set; }
-        public bool Error { get; set; }
+        public enum TokenErrors
+        {
+            ErrorUserNotFound = 404,
+            ErrorExpiredToken = 419
+        }
```

Comment:
Нужно добавить значение указывающее на то что нет ошибок
```suggestion
        public enum LogOnStatus
        {
            Ok = 200,
            ErrorUserNotFound = 404,
            ErrorExpiredToken = 419,
            InternalError = 500
        }
```

Reply:
Добавил